includes/Core/Assets/Assets.php

```diff
@@ -274,6 +274,9 @@ function ( $tag, $handle ) use ( $assets ) {
 	 */
 	private function enqueue_minimal_admin_script() {
 		$this->enqueue_asset( 'googlesitekit_admin' );
+		if ( GOOGLESITEKIT_JS_REFACTOR === true ) {
+			$this->enqueue_asset( 'googlesitekit-api' );
+		}
```

Comment:
What was the reasoning behind this constant? If this is not specifically needed for existing tests we have, then we should remove this from the codebase. The file should be registered like you've done below, but not enqueued. Whenever we'll need it, we can enqueue it, potentially even from another dev utility plugin at first.

Reply:
This was largely to test the file was enqueued and behaving properly, and to give us an easy way to use the new JSR code in the codebase with an option. I figured we could feature-flag the new implementation to allow it to be merged into the codebase sooner than later without actually being used yet...